### DISCUSSION TITLE
Weibull confidence_interval_survival_function_

### DISCUSSION THREAD

**Discussion by armanhak** [CamDavidsonPilon/lifelines#1428] [Ideas]
Hello!
Please add the mathematical formula confidence_interval_survival_function_ for the Weibull model. Unfortunately I didn't find it in the documentation.
I tried to find something myself (https://stats.stackexchange.com/questions/135749/confidence-intervals-of-fitted-weibull-survival-function), the calculation results are similar, but do not match.

**Comment by CamDavidsonPilon** [comment]
Hey @armanhak! Confidence intervals in lifelines are computed with the help of the [delta method](https://en.wikipedia.org/wiki/Delta_method). In the formula for the delta method is a gradient - but lifelines does that gradient computation implicitly using an automatic-differention engine. So, I don't know what the exact formula is and (thankfully) don't need to know for computating CIs!
